posthog/management/commands/generate_demo_events.py

```diff
@@ -0,0 +1,71 @@
+import dataclasses
+import datetime as dt
+import logging
+import secrets
+from itertools import chain
+
+import structlog
+from django.core.management.base import BaseCommand
+
+from posthog.api.capture import capture_internal
+from posthog.demo.products.hedgebox import HedgeboxMatrix
+
+logging.getLogger("kafka").setLevel(logging.WARNING)  # Hide kafka-python's logspam
+
+logger = structlog.get_logger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Generate events using a method that should generate roughly realistic data."
+
+    def add_arguments(self, parser):
+        parser.add_argument("--seed", type=str, help="Simulation seed for deterministic output")
+        parser.add_argument(
+            "--now", type=dt.datetime.fromisoformat, help="Simulation 'now' datetime in ISO format (default: now)"
+        )
+        parser.add_argument(
+            "--days-past",
+            type=int,
+            default=120,
+            help="At how many days before 'now' should the simulation start (default: 120)",
+        )
+        parser.add_argument(
+            "--days-future",
+            type=int,
+            default=30,
+            help="At how many days after 'now' should the simulation end (default: 30)",
+        )
+        parser.add_argument("--n-clusters", type=int, default=500, help="Number of clusters (default: 500)")
+        parser.add_argument(
+            "--team-id", type=str, default="1", help="The team to which the events should be associated."
+        )
```

Comment:
This seems basically the same as `generate_demo_data` and _sounds_ even more similar, so I'm afraid there might be some confusion which is for what. Maybe instead this mode of operation – ingestion via the plugin server instead of directly – could be just a flag on the existing `generate_demo_data` command?

Reply:
Yup 💯 

Still working on this and how it could look, but wanted to avoid modifying the existing command before identifying if it still fits in there.